woocommerce/google-listings-and-ads
# Workaround `woocommerce/data` dependency issues, reset `package-lock.json`

### Changes proposed in this Pull Request:

- Make webpack resolve `[tj]sx` files as well, (1c795ae)
	to workaround `@woocommerce/data` not building after `package-lock.json` reset.

- Add `md5` as dependency, (af2c472)
	as `@woocommerce/data` needs it and does not state it.
	Workaround woocommerce/woocommerce-admin#7086
- Reset `package-lock.json` (06904da)

- Add missing dependencies to `package.json`, (0a09423)
- Add `eslint-import-resolver-webpack` dep to resolve our `.~/` alias for lining (ff4cecd)
- Make `no-unsafe-wp-apis` eslint rule warn iInstead of throwing an error. (236e1c1)
- Add more packages to jest + wc-admin workaround. (0394c06)
	Add `experimental` and all combinations of `package1/node_modules/@woocommerce/package2`.
- Add `@wordpress/core-data` to dependencies. (d4833e4)
	As it is required peer for `@woocommerce/data@1.3.0`.
- Update `@woocommerce/eslint-plugin@1.2.0` dev dependency (1da555c)
	from `^1.0.0-beta.0`, as we already match its requirements.



### Screenshots:

n/a


### Detailed test instructions:

1. Checkout this branch
2. Remove `package-lock.json`
3. install and build
	  ```
	  npm install
	  npm run start
	  ```
4. You should see no build errors.

### Changelog Note:

> Fix dependencies for a clean build after `package-lock.json` reset.
